<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigCreator</class>
 <widget class="QWidget" name="ConfigCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tvConfig">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leTemplateFile">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;path to the input config&lt;/p&gt;&lt;p&gt;eg.: ssh://root@192.168.130.7:config.txt (over ssh)&lt;/p&gt;&lt;p&gt;/tmp/config.txt (locally)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>ssh://root@192.168.130.7:config.cfg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbParse">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>50</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Parse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDeploy">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>130</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Deploy</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leOutputFile">
   <property name="geometry">
    <rect>
     <x>472</x>
     <y>130</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;path to deploy the config to&lt;/p&gt;&lt;p&gt;eg.: ssh://root@192.168.130.7:config.txt (over ssh)&lt;/p&gt;&lt;p&gt;/tmp/config.txt (locally)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>ssh://root@192.168.130.7:config.cfg.mod</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>template</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>110</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Output</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="tbLog">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>330</y>
     <width>291</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbClearLog">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>490</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbTest">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>280</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
